Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -15,6 +15,7 @@ private let KVOLoading = "loading"
 private let KVOEstimatedProgress = "estimatedProgress"
 
 private let HomeURL = "about:home"
+private let URLBeforePathRegex = NSRegularExpression(pattern: "^https?://([^/]+/)", options: nil, error: nil)!
```

Comment:
SVC can't deal with URLBarView directly since it doesn't even know anything about the URL bar (and it shouldn't), so we'd need to get the domain from the SVC protocol, then forward it through BVC to URLBarView. It might be a little funky having the SVC interface returning these domains that it doesn't even use, but that's probably OK for now.

Side note: It's good we kept the SVC delegate separate from the HomePanel delegate, since we'll now be extending it with other functionality!